site_scons/components/__init__.py

```diff
@@ -382,5 +382,14 @@ def define_components(reqs):
                 headers=['protobuf-c/protobuf-c.h'],
                 package='protobuf-c-devel')
 
+    os_name = dist[0].split()[0]
+    if os_name == 'Ubuntu':
+        libcapstone_pkg = 'libcapstone-dev'
+    elif os_name == 'openSUSE':
+        libcapstone_pkg = 'libcapstone-devel'
+    else:
+        libcapstone_pkg = 'capstone-devel'
+    reqs.define('libcapstone', libs=['capstone'], headers=['capstone.h'], package=libcapstone_pkg)
```

Comment:
I would change the name to capstone to be more consistent with other components

Also, this is not sufficient to cause it to check for it. You need to add capstone to the list of client libraries here

https://github.com/daos-stack/daos/blob/47df72b2f4c628208b2cce35c7ed19697da5879b/site_scons/prereq_tools/base.py#L533

Reply:
Thank you! Just fixed as you suggested.